src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/EmbeddedLanguages/VirtualChars/VirtualCharGreenSequence.Chunks.cs

```diff
@@ -106,11 +109,11 @@ public override VirtualChar this[int index]
                 }
 #endif
 
-                var span = new TextSpan(firstVirtualCharPosition + index, length: 1);
+                // var span = new TextSpan(firstVirtualCharPosition + index, length: 1);
```

Comment:
delete

Reply:
Looks to be deleted in the follow up PR https://github.com/dotnet/roslyn/pull/80532/files#diff-20378a3ae3f8efda2a30372129facfa85289aa6afc32f7fab35407e60747a778R99